cmds/coredhcp/config.yml.example

```diff
@@ -10,4 +10,9 @@ server4:
     listen: '0.0.0.0:67'
     interface: "eth0"
     plugins:
-        - server_id: 192.168.1.12
+        # - server_id: 10.10.10.1
+        # - dns: 8.8.8.8 8.8.4.4
+        # - router: 10.10.10.1
+        # - netmask: 255.255.255.0
+        # - file: leases.txt 60s
```

Comment:
Same as above, I would use either `file` or `range` plugin. I would remove this line from the config.